scenes/Asteroid/Asteroid.gd

```diff
@@ -1,7 +1,9 @@
 extends RigidBody2D
 
 onready var animated_sprite = $AnimatedSprite
+onready var asteroid_type = $AnimatedSprite.frames.get_animation_names()
 
 func _ready() -> void:
+	randomize()
 	animated_sprite.frame = rand_range(0,22)
-	animated_sprite.play("Asteroid1")
+	animated_sprite.play(asteroid_type[randi() % asteroid_type.size()])
```

Comment:
I think this would be more clear if we used this instead `randi_range(0, asteroid_type.size() - 1)`

Reply:
Also it seems like the sprite frame gets reset to zero when you make a call to play. I think we should move the line `animated_sprite.frame = rand_range(0,22)` down below this one so that if you have two asteriods with the same animation type then at least they don't start rotating at the same frame.